proto/feature/service.proto

```diff
@@ -38,9 +38,9 @@ import "proto/feature/strategy.proto";
 import "proto/feature/target.proto";
 
 message GetFeatureRequest {
-  string id = 1;
+  string id = 1 [(google.api.field_behavior) = REQUIRED];
```

Comment:
You must also update the api svc for public API.

https://github.com/bucketeer-io/bucketeer/blob/55717f5a7399f5dfe7675baf338078080754f1bb/proto/gateway/service.proto#L151-L153

Reply:
And add the version field.